### PR TITLE
Commerce 2515 Fix option images

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

File: commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jsp [hunk 1]
```diff
@@ -131,6 +131,40 @@ PortalUtil.addPortletBreadcrumbEntry(request, title, StringPool.BLANK, data);
 </div>
 
 <aui:script>
+	function getMetalJsFormData(metalJsForm) {
+		if(!metalJsForm) {
+			return []
```
Comment (ethanbustad) [inline commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jsp]: missing terminal semicolon

File: commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jsp [hunk 2]
```diff
@@ -131,6 +131,40 @@ PortalUtil.addPortletBreadcrumbEntry(request, title, StringPool.BLANK, data);
...
+
+		return Object.values(renderer.refs)
+			.map(
+				function(option) {
+					return {
+						key: option.fieldName,
+						value: option.value
+					}
+				}
+			)
```
Comment (ethanbustad) [inline commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jsp]: missing terminal semicolon here, too

File: commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jsp [hunk 3]
```diff
@@ -131,6 +131,40 @@ PortalUtil.addPortletBreadcrumbEntry(request, title, StringPool.BLANK, data);
...
+			}
+			ddmForm.on(
+				'fieldEdited',
+				function() {
+					var fieldValues = getMetalJsFormData(ddmForm);
+
+					var form = AUI.$(document.<portlet:namespace />fm);
+					form.fm('ddmFormValues').val(JSON.stringify(fieldValues));
+				}
+			)
```
Comment (ethanbustad) [inline commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jsp]: and here

File: commerce-product-definitions-web/src/main/resources/META-INF/resources/instance/details.jsp [hunk 4]
```diff
@@ -168,6 +168,40 @@ productSkusURL.setParameter("screenNavigationCategoryKey", cpInstanceDisplayCont
 </aui:form>
 
 <aui:script>
+	function getMetalJsFormData(metalJsForm) {
+		if(!metalJsForm) {
+			return []
```
Comment (ethanbustad) [inline commerce-product-definitions-web/src/main/resources/META-INF/resources/instance/details.jsp]: same missing semicolons in this file

**Review by ethanbustad — APPROVED** [review]

**Comment by ethanbustad** [comment]
ci:test

File: commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jsp [hunk 5]
```diff
@@ -18,7 +18,7 @@
 
 <%
 CPAttachmentFileEntriesDisplayContext cpAttachmentFileEntriesDisplayContext = (CPAttachmentFileEntriesDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
-
+f
```
Comment (ethanbustad) [inline commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jsp:21]: f

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by ethanbustad** [comment]
Sent along to https://github.com/liferay/com-liferay-commerce/pull/1785 after removing the `f` :)

**Pull request closed without merging**